main.go

```diff
@@ -48,7 +48,11 @@ func generateFake(interfaceName, sourcePackageDir, importPath, outputPath, desti
 	var err error
 	var iface *model.InterfaceToFake
 	if sourcePackageDir == "" {
-		iface, err = locator.GetInterfaceFromImportPath(interfaceName, importPath)
+		var vendorPaths []string
+		vendorPaths, err = locator.VendorPathsForDirPath(".")
```

Comment:
This looks like this could call to check for vendor paths could easily done inside the call to `GetInterfaceFromImportPath`. I think that would simplify this code here in `main.go`, which we should absolutely do, since it's not tested by unit tests.

In general, if you're calling multiple methods to an object (or in this case, a package) and then passing the results from one directly into the other, it's a case of the abstraction you've created leaking, and not having a well defined interface for how to use it.